src/_pytest/python.py

```diff
@@ -143,9 +144,29 @@ def async_fail(nodeid: str) -> None:
     fail(msg, pytrace=False)
 
 
+def async_default_arg_warn(nodeid: str, function_name, param) -> None:
```

Comment:
Why `async_`? What's async about it?

Reply:
I agree that I shouldn't have added async_ to the function name. I'll update that.